src/containers/Board/Board.js

```diff
@@ -152,7 +154,7 @@ export class Board extends Component {
   };
 
   handleAddClick = () => {
-    this.setState({ symbolDetailsOpen: true });
+    this.setState({ symbolDetailsOpen: true, selectedSymbols: [], isSelecting: false })
```

Comment:
I think it's better to cancel the selecting mode when adding a new symbol.
Several times I tried to play the newly added symbol without noticing I was still in select mode

Reply:
totally agree!